src/Components/Nav/TopNav.jsx

```diff
@@ -27,22 +28,26 @@ function TopNav() {
 
   return (
     <div className="top-nav">
+      <div className="title">
+        <h1 className="text">{t("global.vocascan")}</h1>
+      </div>
       <h3 className="user" onClick={handleClick}>
+        <PersonIcon />
```

Comment:
Icon is not in in line with Text
![Unbenannt](https://user-images.githubusercontent.com/54618409/112480991-d9859600-8d76-11eb-9e9c-8f1fe358432c.PNG)

Reply:
ich hab noch die top nav vom Verhalten her angepasst.